web-client/src/presenter/computeds/formattedEligibleCasesHelper.js

```diff
@@ -1,42 +1,47 @@
-import { setConsolidationFlagsForDisplay } from '../../../../shared/src/business/utilities/setConsolidationFlagsForDisplay';
+import {
+  formatCase,
+  getSortableDocketNumber,
+} from '../../../../shared/src/business/utilities/getFormattedTrialSessionDetails';
 import { state } from 'cerebral';
 
-const compareTrialSessionEligibleCases = eligibleCases => (a, b) => {
-  if (a.isManuallyAdded && !b.isManuallyAdded) {
-    return -1;
-  } else if (!a.isManuallyAdded && b.isManuallyAdded) {
-    return 1;
-  } else if (a.highPriority && !b.highPriority) {
-    return -1;
-  } else if (!a.highPriority && b.highPriority) {
-    return 1;
-  } else if (a.isDocketSuffixHighPriority && !b.isDocketSuffixHighPriority) {
-    return -1;
-  } else if (!a.isDocketSuffixHighPriority && b.isDocketSuffixHighPriority) {
-    return 1;
-  } else if (
-    (a.isManuallyAdded && b.isManuallyAdded) ||
-    (a.highPriority && b.highPriority) ||
-    (a.isDocketSuffixHighPriority && b.isDocketSuffixHighPriority)
-  ) {
-    let aSortString = getSortableDocketNumber(a.docketNumber);
-    let bSortString = getSortableDocketNumber(b.docketNumber);
-    return aSortString.localeCompare(bSortString);
-  } else {
-    let aSortString = getFullSortString(a, eligibleCases);
-    let bSortString = getFullSortString(b, eligibleCases);
-    return aSortString.localeCompare(bSortString);
-  }
-};
+const compareTrialSessionEligibleCases =
+  ({ eligibleCases }) =>
+  (a, b) => {
+    if (a.isManuallyAdded && !b.isManuallyAdded) {
+      return -1;
+    } else if (!a.isManuallyAdded && b.isManuallyAdded) {
+      return 1;
+    } else if (a.highPriority && !b.highPriority) {
+      return -1;
+    } else if (!a.highPriority && b.highPriority) {
+      return 1;
+    } else if (a.isDocketSuffixHighPriority && !b.isDocketSuffixHighPriority) {
+      return -1;
+    } else if (!a.isDocketSuffixHighPriority && b.isDocketSuffixHighPriority) {
+      return 1;
+    } else if (
+      (a.isManuallyAdded && b.isManuallyAdded) ||
+      (a.highPriority && b.highPriority) ||
+      (a.isDocketSuffixHighPriority && b.isDocketSuffixHighPriority)
+    ) {
+      let aSortString = getSortableDocketNumber(a.docketNumber);
+      let bSortString = getSortableDocketNumber(b.docketNumber);
+      return aSortString.localeCompare(bSortString);
+    } else {
+      let aSortString = getEligibleDocketNumberSortString({
+        allCases: eligibleCases,
+        theCase: a,
+      });
+      let bSortString = getEligibleDocketNumberSortString({
+        allCases: eligibleCases,
+        theCase: b,
+      });
+      return aSortString.localeCompare(bSortString);
+    }
+  };
 
-const getSortableDocketNumber = docketNumber => {
-  const [number, year] = docketNumber.split('-');
-  return `${year}-${number.padStart(6, '0')}`;
-};
-
-const getFullSortString = (theCase, cases) => {
-  const leadCase = cases.find(
+const getEligibleDocketNumberSortString = ({ allCases, theCase }) => {
```

Comment:
I know you didn't make this variable, but is there a better name we can use for "theCase"?